build.cmd

```diff
@@ -272,7 +274,15 @@ if defined __SkipTestBuild (
 echo.
 echo Commencing build of tests for %__BuildOS%.%__BuildArch%.%__BuildType%
 echo.
+
+if defined _priorityvalue (
+    call "%__ProjectDir%\tests\buildtest.cmd" Priority %_priorityvalue%
+    goto ErrCheck
```

Comment:
I would make a variable to hold buildtest args, and if _priorityvalue is defined, add the priority stuff to it. That will allow you to remove this goto and have just one place where you call buildtest.